sockeye/train.py

```diff
@@ -1026,9 +1026,7 @@ def train(args: argparse.Namespace, custom_metrics_logger: Optional[Callable] =
                                            max_seq_len_target=max_seq_len_target,
                                            config_data=config_data)
 
-        training_model = model.SockeyeModel(
-            model_config,
-            train_decoder_only=args.fixed_param_strategy == C.FIXED_PARAM_STRATEGY_ALL_EXCEPT_DECODER)
+        training_model = model.SockeyeModel(model_config)
```

Comment:
It seems like there are some recent changes that show as 'deleted'. Could you merge from master to make sure no recent changes get reverted?